manifesto/personas.md

```diff
@@ -34,30 +34,34 @@ The following personas are fictional characters that we defined in order to repr
+| Kubernetes experience (1-6)       | :book: :book: :book: :book:                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| Accountability/Decisiveness (1-6) | 💪 💪 💪 💪 💪 💪                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| Slogans                           | _I know what I want... not yet sure how to achieve it._                                                                                                                                                                                                                                                                                                                                                                                                   |
 
 ### GAINS
 
-- :green_heart: I need a comprehensive set of opinionated tools (CNCF-approved, recognized by the community).
-- :green_heart: I need certainty that Kyma uses industry standards.
+- :green_heart: I need to evaluate Kyma fast (within 1 day).
+- :green_heart: I love tutorials, sample code, recepies that refer to my use cases (i.e authentication/authorisation flows).
+- :green_heart: I want to tick off on product (industry) standards for free (i.e security, accessibility, observability, tracing, etc).
+- :green_heart: I want future proof platform - based on references.
+- :green_heart: With Kyma I want to connect to systems I know (public and internal systems).
 - :green_heart: I need a trial version of Kyma before I can recommend it.
-- :green_heart: I don't want to take extra care about security/monitoring/tracing...
-- :green_heart: I need developers in my organization to be focused on the solutions rather than infrastructure.
+- :green_heart: I need agility in architecture designs - I want to replace individual modules if needed.
+- :green_heart: I want a painless upgrade path.
+- :green_heart: I want operational support.
 
 ### PAINS
 
-- :broken_heart: I'm missing success stories, references.
-- :broken_heart: I don't want my dev teams to struggle.
-- :broken_heart: I'm missing best practices, recipes of how to put things together.
+- :broken_heart: I'm giving up on a project if success stories, references are missing
```

Comment:
```suggestion
- :broken_heart: I give up on a project if success stories and/or references are missing.
```
`I'm giving up` would mean that he's doing it now. I believe what you mean here is a general rule for him.